.github/workflows/ci-deploy-env.yml

```diff
@@ -79,28 +80,19 @@ jobs:
           --tag $ECR_REGISTRY/$ECR_REPOSITORY:$BRANCH \
           --file ./Dockerfile ./
 
-#    - name: Kick off Terraform deploy in sysops/ if not live
-#      id: sysops-deploy
-#      run: |
-#        BRANCH=$(echo "${GITHUB_REF#refs/*/}" | sed 's/.*\///')
-#        if [[ "$BRANCH" != master ]]; then
-#          curl \
-#          -X POST \
-#          -H "Accept: application/vnd.github.v3+json" \
-#          https://api.github.com/repos/meedan/sysops/actions/workflows/deploy_${{ github.event.repository.name }}.yml/dispatches \
-#          -u ${{ secrets.SYSOPS_RW_GITHUB_TOKEN }} \
-#          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "branch": "'$BRANCH'"}}'
-#        fi
-
-#   - name: Send Slack Notification
-#     id: slack-notify
-#     if: ${{ always() }}
-#     uses: 8398a7/action-slack@v3
-#     with:
-#       status: ${{ job.status }}
-#       author_name: 'Github Actions'
-#       icon_emoji: ':octocat:'
-#       fields: repo,message,author # selectable (default: repo,message)
-#     env:
-#       GITHUB_TOKEN: ${{ secrets.CONFIGURATOR_TOKEN }} # optional
-#       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }} # required
+  # trigger-sysops-deploy:
+  #   runs-on: ubuntu-latest
+  #   needs: [build-push]
+  #   steps:
```

Comment:
what is this additional job for?

Reply:
I think this is answered here: https://github.com/meedan/pender/pull/539/files#r2054911625

Reply:
I see. while this is okay, it's not necessary. creating a new job in a workflow spins up a new container that needs to be initialized, with everything installed and tokens added. it normally will also cost money to spin up, but since pender is an open source repo that's not relevant in this case. it is a lot of infrastructural overhead just to send an API call though!